COMMANDS.md

```diff
@@ -1,35 +1,35 @@
 # Command Reference
 
 Like `git`, the `goenv` command delegates to subcommands based on its
-first argument. 
+first argument.
 
 All subcommands are:
 
-* [`goenv commands`](#goenv-commands)
-* [`goenv completions`](#goenv-completions)
-* [`goenv exec`](#goenv-exec)
-* [`goenv global`](#goenv-global)
-* [`goenv help`](#goenv-help)
-* [`goenv hooks`](#goenv-hooks)
-* [`goenv init`](#goenv-init)
-* [`goenv install`](#goenv-install)
-* [`goenv local`](#goenv-local)
-* [`goenv prefix`](#goenv-prefix)
-* [`goenv rehash`](#goenv-rehash)
-* [`goenv root`](#goenv-root)
-* [`goenv shell`](#goenv-shell)
-* [`goenv shims`](#goenv-shims)
-* [`goenv uninstall`](#goenv-uninstall)
-* [`goenv version`](#goenv-version)
-* [`goenv --version`](#goenv---version)
-* [`goenv version-file`](#goenv-version-file)
-* [`goenv version-file-read`](#goenv-version-file-read)
-* [`goenv version-file-write`](#goenv-version-file-write)
-* [`goenv version-name`](#goenv-version-name)
-* [`goenv version-origin`](#goenv-version-origin)
-* [`goenv versions`](#goenv-versions)
-* [`goenv whence`](#goenv-whence)
-* [`goenv which`](#goenv-which)
+- [`goenv commands`](#goenv-commands)
+- [`goenv completions`](#goenv-completions)
+- [`goenv exec`](#goenv-exec)
+- [`goenv global`](#goenv-global)
+- [`goenv help`](#goenv-help)
+- [`goenv hooks`](#goenv-hooks)
+- [`goenv init`](#goenv-init)
+- [`goenv install`](#goenv-install)
+- [`goenv local`](#goenv-local)
+- [`goenv prefix`](#goenv-prefix)
+- [`goenv rehash`](#goenv-rehash)
+- [`goenv root`](#goenv-root)
+- [`goenv shell`](#goenv-shell)
+- [`goenv shims`](#goenv-shims)
+- [`goenv uninstall`](#goenv-uninstall)
+- [`goenv version`](#goenv-version)
+- [`goenv --version`](#goenv---version)
+- [`goenv version-file`](#goenv-version-file)
+- [`goenv version-file-read`](#goenv-version-file-read)
+- [`goenv version-file-write`](#goenv-version-file-write)
+- [`goenv version-name`](#goenv-version-name)
+- [`goenv version-origin`](#goenv-version-origin)
+- [`goenv versions`](#goenv-versions)
+- [`goenv whence`](#goenv-whence)
+- [`goenv which`](#goenv-which)
```

Comment:
2cents @ChronosMasterOfAllTime - it makes for harder to review PRs, if in each there are changes that are unrelated from the focus of the PR.

I totally understand how tempting it is, but I'd try to avoid it, especially if it's stuff like "should I create a list in MD with `*` or `-`", that is totally inconsequential at rendering.

Reply:
Yeah sorry about that. I had my formatters turned on when saving so it automatically removes whitespace etc...will fix the MD changes

Reply:
Fixed